pages/docs/components/overview.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  GridItem,
+  Heading,
+  List,
+  ListItem,
+  SimpleGrid,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { GetStaticProps } from 'next'
+
+import componentsSidebar from 'configs/components-sidebar.json'
```

Comment:
What do you think about moving the sidebar json files to ts files ? We could ensure way better typing through the complete application. We could also do this in a separate ticket but I saw you made your own types here.

Reply:
Yeah that's actually a good idea! I'll see all the dependencies of the components sidebar json file and do a little refactor.